examples/src/main/scala/org/apache/spark/examples/ml/ALSExample.scala

```diff
@@ -65,8 +65,6 @@ object ALSExample {
 
     // Evaluate the model by computing the RMSE on the test data
     val predictions = model.transform(test)
-      .withColumn("rating", col("rating").cast(DoubleType))
```

Comment:
Thanks will remove them

On Wed, 4 May 2016 at 17:17, Seth Hendrickson <notifications@github.com>
wrote:

> In examples/src/main/scala/org/apache/spark/examples/ml/ALSExample.scala
> <https://github.com/apache/spark/pull/12892#discussion_r62057972>:
>
> > @@ -65,8 +65,6 @@ object ALSExample {
> >
> >      // Evaluate the model by computing the RMSE on the test data
> >      val predictions = model.transform(test)
> > -      .withColumn("rating", col("rating").cast(DoubleType))
>
> You have some unused imports at the top of this file now. Don't need
>
> import org.apache.spark.sql.functions._import org.apache.spark.sql.types.DoubleType
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/12892/files/a591564a6f66b8461ddccfe979a6aa1976ecdebe#r62057972>
>